quantum/audio/song_list.h

```diff
@@ -1,4 +1,7 @@
-/* Copyright 2016 Jack Humbert
+/* Any song or sound without a license explicitly stated is:
+ *
+ * Copyright 2016 Jack Humbert
+ * Copyright 2019 Zach White
```

Comment:
@skullydazed committed `CLUEBOARD_SOUND` in November of 2017.

```suggestion
 * Copyright 2017 Zach White
```